Gemfile

```diff
@@ -1,6 +1,7 @@
 source "https://rubygems.org"
 
-gem "middleman-core", git: "https://github.com/middleman/middleman.git"
+# Use a specific version compatible with Ruby 2.7.2
+gem "middleman-core", "~> 4.3.11"
```

Comment:
Middleman already supports Ruby 2.7, I think we should revert this change.

Reply:
No problemo. Addressed in ca12c202133d6e39590fe4a918490f72f6a600e7